doc/lua_api.txt

```diff
@@ -1797,8 +1801,17 @@ Example:
 Formspec
 ========
 
-Formspec defines a menu. Currently not much else than inventories are
-supported. It is a string, with a somewhat strange format.
+Formspec defines a menu. This supports inventories and some of the
+typical widgets like buttons, checkboxes, text input fields, etc.
+It is a string, with a somewhat strange format.
+
+A formspec is made out of formspec elements, which includes widgets
+like buttons but also includes some metadata about stuff like background
```

Comment:
Metadata is confusing here, because it's used to describe the string key/value storage in the Minetest docs in many places.